src/test/java/com/hazelcast/wm/test/DeferredWriteClusterTest.java

```diff
@@ -32,7 +32,7 @@
 
 import java.io.File;
 
-import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.*;
```

Comment:
We don't use `*` import, can you please add separate imports for each import you need?